erpnext/accounts/doctype/purchase_invoice/purchase_invoice.js

```diff
@@ -721,6 +721,14 @@ frappe.ui.form.on("Purchase Invoice", {
 					if (response) frm.set_value("credit_to", response.message);
 				},
 			});
+
+			frappe.call({
```

Comment:
I think this should be added in the buying controller because it is not implemented in the purchase order or purchase invoice